utils/forest_utils/src/db/file_backed_obj.rs

```diff
@@ -1,36 +1,61 @@
 // Copyright 2019-2023 ChainSafe Systems
 // SPDX-License-Identifier: Apache-2.0, MIT
 
-use std::{path::PathBuf, str::FromStr};
+use std::{
+    path::PathBuf,
+    str::FromStr,
+    time::{Duration, SystemTime},
+};
 
+use ahash::HashSet;
 use cid::Cid;
 use log::warn;
 
 pub struct FileBacked<T: FileBackedObject> {
     inner: T,
     path: PathBuf,
+    last_sync: Option<SystemTime>,
 }
 
+const SYNC_PERIOD: Duration = Duration::from_secs(600);
```

Comment:
Shall we add it to the configuration?

Reply:
I'm not sure if it's really critical for our users to be configurable. I prefer to leave it like this for now (ten minutes should be a good start).

Reply:
If you make it configurable then you will be able to add some unit tests. :)

Reply:
But modifying `last_sync` directly is also fine.

Reply:
I've made it configurable.